examples/acquiring_data/fido_metadata_queries.py

```diff
@@ -0,0 +1,57 @@
+"""
+====================================
+Querying Metadata clients using Fido
+====================================
+
+This example shows how to search and retrieve metadata using `~sunpy.net.Fido` from
+search facilities like `~sunpy.net.hek.HEKClient`, `~sunpy.net.helio.HECClient`,
+and `~sunpy.net.jsoc.JSOCClient`. It also shows how to display desired columns from the result.
+"""
+import os
+
+from sunpy.net import Fido
+from sunpy.net import attrs as a
+
+###################################################################
+# We will query Helio for the 'rhessi_flare_list' table and
+# limit the total number of records to 5.
+# For the same time range, we will query HEK for 'FL' as the
+# event type and 'PeakFlux' greater than 1000.
+# We will also search JSOC for 'hmi.m_45s' series.
+
+timerange = a.Time('2010/8/1 03:40', '2010/8/1 3:40:10')
+# Note that JSOC needs an email address that you should manually supply here.
+results = Fido.search(timerange, a.helio.TableName('rhessi_hxr_flare') |
+                      a.hek.FL & (a.hek.FL.PeakFlux > 1000) |
+                      a.jsoc.Series('hmi.m_45s') & a.jsoc.Notify(os.environ["JSOC_EMAIL"]))
```

Comment:
I suggest we set `jsoc_email = os.environ["JSOC_EMAIL"])` as a variable before this call with an explanation to the reader of what they should override it to be when running the example.

Reply:
Fixed.